src/Swarm/Language/Syntax.hs

```diff
@@ -426,79 +433,131 @@ isUserFunc c = case constMeta $ constInfo c of
+      ]
+  Place ->
+    commandLow 1 . doc "Place an item at the current location." $
+      ["The current location has to be empty for this to work."]
+  Give -> commandLow 2 "Give an item to another robot closeby."
+  Install -> commandLow 2 "Install a device from inventory on a robot."
+  Make -> commandLow 1 "Make an item using a recipe."
+  Has -> commandLow 1 "Sense whether the robot has a given item in its inventory."
+  Count -> commandLow 1 "Get the count of a given item in robots inventory."
+  Reprogram ->
+    commandLow 2 . doc "Reprogram another robot with new command." $
+      ["The other robot has to be closeby and idle."]
+  Drill ->
+    commandLow 1 . doc "Drill through an entity." $
+      [ "Usually you want to 'drill forward' when exploring to clear out obstacles."
+      , "When you have found a source to drill, you can stand on it and 'drill down'."
+      , "See what recipes with drill you have available."
+      ]
+  Build ->
+    commandLow 1 . doc "Construct a new robot." $
+      [ "You can specify a command for the robot to execute."
+      , "If the command requires a device it will be installed from your inventory."
+      ]
+  Salvage ->
+    commandLow 0 . doc "Deconstruct an old robot." $
+      ["Salvaging a robot will give you its inventory, installed devices and log."]
+  Say ->
+    commandLow 1 . doc "Emit a message." $ -- TODO: #513
+      [ "The message will be in global log, which you can not currently view."
+      , "https://github.com/swarm-game/swarm/issues/513"
+      ]
+  Log -> commandLow 1 "Log the string in the robots logger."
+  View -> commandLow 1 "View the given robot."
+  Appear ->
+    commandLow 1 . doc "Set how the robot is displayed." $
+      [ "You can either specify one character or five (for each direction)."
+      , "The default is \"X^>v<\"."
+      ]
+  Create ->
+    commandLow 1 . doc "Create an item out of thin air." $
+      ["Only available in creative mode."]
+  Whereami -> commandLow 0 "Get the current x, y coordinates."
+  Blocked -> commandLow 0 "See if the robot can move forward."
+  Scan ->
+    commandLow 0 . doc "Scan a nearby location for entities." $
+      [ "Adds the entity (not robot) to your inventory with count 0 if there is any."
+      , "If you can use sum types, you can also inspect the result directly."
+      ]
+  Upload -> commandLow 1 "Upload robots known entities to another robot."
+  Ishere -> commandLow 1 "See if a specific entity is in current location."
+  Self -> functionLow 0 "Get a reference to current robot."
+  Parent -> functionLow 0 "Get a reference to the robot's parent."
+  Base -> functionLow 0 "Get a reference to the base robot."
+  Whoami -> commandLow 0 "Get the robot's display name."
+  Setname -> commandLow 1 "Set the robot's display name."
+  Random ->
+    commandLow 1 . doc "Get a uniformly random integer." $
+      ["The random integer will be from the range 0 to END inclusive of your choice."]
```

Comment:
```suggestion
      ["The random integer will be chosen from the range 0 to n-1, exclusive of the argument."]
```

Reply:
I briefly checked the code, but I should have tried it in REPL instead. :sweat_smile: